src/cdk/collections/recycle-view-repeater-strategy.ts

```diff
@@ -37,10 +37,10 @@ import {
 export class _RecycleViewRepeaterStrategy<T, R, C extends _ViewRepeaterItemContext<T>>
     implements _ViewRepeater<T, R, C> {
   /**
-   * The size of the cache used to store unused views.
-   * Setting the cache size to `0` will disable caching. Defaults to 20 views.
+   * The size of the cache used to store unused views. Setting the cache size to `0` will disable
+   * caching. Defaults to 100 views.
```

Comment:
Can you add some justification of the default? I think I chose 20 at random at some point, but since this change probably has a little more intent behind it we should document that

Reply:
Tables tend to render more rows than the average virtual scroll viewport. Increasing the number reduces some of the header flickering. The default cache size for the table should probably be independent of the general purpose strategy.

 I decided to revert this file.